app/pages/lab-pages-editor/components/TasksPage/TasksPage.jsx

```diff
@@ -124,6 +149,14 @@ export default function TasksPage() {
           >
             LINK
           </button>
+          <button
+            className="big"
+            onClick={editStep}
```

Comment:
Noting this doesn't send an active step to the editStep handler to this button open an empty EditStepDialog atm.